libr/include/r_util/r_sys.h

```diff
@@ -29,6 +29,8 @@ typedef struct _os_info {
 	int compilation;
 } os_info;
 
+R_API int r_sys_sigaction(int sig, void (*handler) (int));
+R_API int r_sys_sigaction_mask(int sig, void (*handler) (int), sigset_t mask);
```

Comment:
`sigset_t` is platform-specific. These functions should be inside an `#if __UNIX__`